.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__ . '/example')
+    ->in(__DIR__ . '/src')
+    ->in(__DIR__ . '/test')
+;
+
+$config = new PhpCsFixer\Config();
+return $config->setRules([
+    '@Symfony' => true,
+    'array_syntax' => ['syntax' => 'short'],
```

Comment:
Tutaj właśnie zastanawiałbym się czy to konieczne z tym `array_syntax`. https://cs.symfony.com/doc/rules/array_notation/array_syntax.html - rozumiem tak, że jeżeli mamy `Symfony` to nie musimy :D

Reply:
zatem usunąłem `array_syntax`